spec/models/post_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  post = Post.new
+
+  it 'should not create post because title is not given' do
+    post.title = nil
+    expect(post).to_not be_valid
+  end
+  it 'should not accecpt a values for title over 250 chars' do
+    post.title = 'A very long title' * 20
+    expect(post).to_not be_valid
+  end
+  it 'should not accecpt negative values for comments counter' do
+    post.comments_counter = -3
+    expect(post).to_not be_valid
+  end
+  it 'should not accecpt negative values for likes counter' do
+    post.likes_counter = -20
+    expect(post).to_not be_valid
+  end
+end
```

Comment:
- [ ] All custom methods in each model should be tested. Kindly add tests for the `five_recent_comments` which you have in the Post model.